src/pages/Product/components/ColorButton/ColorButton.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import '../ColorButton/ColorButton.scss'
+
+const ColorButton = (props) => {
+    const {color, setColor} = props
+    return (
+        <>
+            <button className={`colorButton ${color}`} onClick={() => {setColor(color)}} />
+        </> )
```

Comment:
```suggestion
            <button className={`colorButton ${color}`} onClick={() => {setColor(color)}} />
)
```

버튼 요소가 하나기에 꼭 fragment 태그로 감싸줄 필요는 없습니다.